src/QMCHamiltonians/CoulombPBCAA.cpp

```diff
@@ -415,13 +415,13 @@ CoulombPBCAA::Return_t CoulombPBCAA::evalConsts(bool report)
   }
   if (report)
     app_log() << "   PBCAA self-interaction term " << Consts << std::endl;
-  //Compute Madelung constant: this is not correct for general cases
+  //Neutraling background term
+  mRealType vs_k0 = AA->evaluateSR_k0(); //v_s(k=0)
+  //Compute Madelung constant
   MC0 = 0.0;
   for (int i = 0; i < AA->Fk.size(); i++)
     MC0 += AA->Fk[i];
-  MC0 = 0.5 * (MC0 - vl_r0);
-  //Neutraling background term
-  mRealType vs_k0 = AA->evaluateSR_k0(); //v_s(k=0)
+  MC0 = 0.5 * (MC0 - vl_r0 - vs_k0);
```

Comment:
As I recall, MC0 is only used for reporting (print to stdout).  As such, the Ewald value returned by CoulombPBCAA::evaluate() is unchanged, correct?

Reply:
yes